Falha ao compilar o front-end da aplicação

Atualmente, o processo de compilação do frontend da aplicação falha com as seguintes mensagens de erro:

`18.94 ✘ [ERROR] The 'pontos/editar/:id' route uses prerendering and includes parameters, but 'getPrerenderParams' is missing. Please define 'getPrerenderParams' function for this route in your server routing configuration or specify a different 'renderMode'.`

`18.94 ✘ [ERROR] The 'pontos/:id' route uses prerendering and includes parameters, but 'getPrerenderParams' is missing. Please define 'getPrerenderParams' function for this route in your server routing configuration or specify a different 'renderMode'.`

--- a/Front-end/zarpar/src/app/app.routes.server.ts
+++ b/Front-end/zarpar/src/app/app.routes.server.ts
@@ -2,6 +2,14 @@ import { RenderMode, ServerRoute } from '@angular/ssr';
 
 export const serverRoutes: ServerRoute[] = [
   {
+    path: 'pontos/:id',
+    renderMode: RenderMode.Server
+  },
+  {
+    path: 'pontos/editar/:id',
+    renderMode: RenderMode.Server
+  },
+  {
     path: '**',
     renderMode: RenderMode.Prerender
   }

